Generator/Templates/GObject/class.sbntxt

```diff
@@ -67,6 +67,22 @@ namespace {{ namespace }}
         #endregion
 
         {{~ end ~}}
+        
+        #region Fields
+                
+        [StructLayout(LayoutKind.Sequential)]
+        protected {{'new'|get_if_parent}} struct Fields : IObjectStruct
+        {        
+{{ # TODO: Do we want to use field structs or put the fields in objects directly? 
```

Comment:
I like the `Fields` struct :+1: Remove the TODO?

Reply:
Fixed